source/documentation/contribute-to-rdoproject.html.md

```diff
@@ -0,0 +1,92 @@
+---
+author: chandankumar
+title: Let's Contribute to RDO Project
+---
+
+# Let's Contribute to RDO Project
+
+So contributing to RDO Project is hard, I think no! we need to follow some steps
```

Comment:
s/is hard/can look challenging for beginners/ 
hard has a negative connotation, let's rephrase it.